packages/ckeditor5-word-count/tests/wordcount.js

```diff
@@ -16,7 +16,7 @@ import Position from '@ckeditor/ckeditor5-engine/src/model/position';
 import ShiftEnter from '@ckeditor/ckeditor5-enter/src/shiftenter';
 import TableEditing from '@ckeditor/ckeditor5-table/src/tableediting';
 import env from '@ckeditor/ckeditor5-utils/src/env';
-import ListEditing from '@ckeditor/ckeditor5-list/src/listediting';
+import ListEditing from '@ckeditor/ckeditor5-list/src/list/listediting';
```

Comment:
Don't forget to mention that for integrators this is a BC. They could be using this feature with their own UI and after bumping the editor version they will have imports pointing to nothing.